cmd/root.go

```diff
@@ -67,6 +67,10 @@ or ./scorecard --{npm,pypi,rubgems}=<package_name> [--checks=check1,...] [--show
 	Short: "Security Scorecards",
 	Long:  "A program that shows security scorecard for an open source software.",
 	Run: func(cmd *cobra.Command, args []string) {
+		if token, exists := os.LookupEnv("GITHUB_AUTH_TOKEN"); !exists || token == "" {
+			log.Fatalf("GITHUB_AUTH_TOKEN env var is not set. Please set this to your Github PAT before running this command.")
```

Comment:
Nice, maybe link to https://github.com/ossf/scorecard#authentication (or https://docs.github.com/en/free-pro-team@latest/developers/apps/about-apps#personal-access-tokens) as well?

Reply:
Yes, we should. @laurentsimon FYI.